alica_engine/include/engine/AlicaContext.h

```diff
@@ -533,7 +546,7 @@ template <class TraceFactoryType, class... Args>
 void AlicaContext::setTraceFactory(Args&&... args)
 {
     if (_initialized) {
-        ALICA_WARNING_MSG("AC: Context already initialized. Can not set new tracefactory");
+        Logging::LoggingUtil::logWarn() << "AC: Context already initialized. Can not set new tracefactory";
```

Comment:
Can we take the log "namespace" (in this case the `AC` part) and have it as an optional argument to `logWarn()`?
Then we can later configure things like are available in ROS logger:
- Filter logs of certain namespaces based on yaml config
- Configure displaying/not displaying the namespace
- Configure order and show namespace e.g. before the timestamp

Reply:
can do, but will require adding a `std::string namespace` argument to the `log` interface in `IAlicaLogger`